src/status_im2/contexts/quo_preview/drawers/action_drawers.cljs

```diff
@@ -16,58 +16,47 @@
    {:label "Show red options?"
     :key   :show-red-options?
     :type  :boolean}
-   {:label   "Override theme"
-    :key     :override-theme
+   {:label   "Drawer theme"
```

Comment:
hmm, let's just remove this option and take it from the system settings. @mmilad75 and I were discussing we could also put the system theme selector in the preview screens too? rather than in the preview menu as it makes it easier to experiment with.

Reply:
@J-Son89 - I think we will need this selector as it will help us test the scenario such as the user's screen in light mode, but the bottom sheet needs to be rendered in dark mode.

This is the same as how the bottom sheet is used in Activity Center.